pkg/dependency/parser/sbt/lockfile/parse.go

```diff
@@ -0,0 +1,77 @@
+package lockfile
+
+import (
+	"io"
+
+	"github.com/liamg/jfather"
+	"golang.org/x/xerrors"
+
+	"github.com/aquasecurity/trivy/pkg/dependency"
+	ftypes "github.com/aquasecurity/trivy/pkg/fanal/types"
+	xio "github.com/aquasecurity/trivy/pkg/x/io"
+)
+
+// lockfile format defined at: https://stringbean.github.io/sbt-dependency-lock/file-formats/version-1.html
+type sbtLockfile struct {
+	Version        int                     `json:"lockVersion"`
+	Timestamp      string                  `json:"timestamp"`
+	Configurations []string                `json:"configurations"`
+	Dependencies   []sbtLockfileDependency `json:"dependencies"`
+}
+
+type sbtLockfileDependency struct {
+	Organization   string                `json:"org"`
+	Name           string                `json:"name"`
+	Version        string                `json:"version"`
+	Artifacts      []sbtLockfileArtifact `json:"artifacts"`
+	Configurations []string              `json:"configurations"`
```

Comment:
We include only `runtime` and `compile` scopes for `pom.xml` files:
https://github.com/aquasecurity/trivy/blob/6e7f62d2debbb4acbeb7ad58ecd8ac7379f19e40/pkg/dependency/parser/java/pom/parse.go#L396-L398

I think we need to use same logic here.